Makefile

```diff
@@ -167,6 +167,17 @@ endif
 # Fix permissions and ownership for password files:
 	-$(INSTALL_USER) -m 0600 -t $(DESTDIR)$(judgehost_etcdir) \
 		etc/restapi.secret
+	@echo ""
+	@echo "========== Judgehost Install Completed =========="
+	@echo ""
+	@echo "Optionally:"
+	@echo "    - Install the create-cgroup service to setup the secure judging restrictions:"
+	@echo "        cp <DOMSERVER_UNPACK_PATH>/judge/create-cgroups.service /etc/systemd/system/"
```

Comment:
In other places we don't refer to the base path of the installation. Also this is normally run from that directory, so I'd remove the `<DOMSERVER_UNPACK_PATH>/`, also in other places in this PR.